lib/Services/cayenne.js

```diff
@@ -145,6 +155,16 @@ class Cayenne {
     return this.auth.send(this.service, 'GET', url, { query });
   }
 
+  /**
+   * Destroys all groups and attributes for a user
+   * @param {String} userId The user id of the user that owns the parent device
+   */
+  destroyGroups(userId) {
```

Comment:
Nitpick, can we have consistent naming for method? As all others are `deleteX`, rename to `deleteGroups`